src/Info.elm

```diff
@@ -119,65 +119,61 @@ placeholderInfo =
 info : List Info
 info =
     [ { id = 1
+      , name = "About the project"
+      , slug = "about"
+      , icon = "question-circle-o"
+      , infoText =
+            [ """In 2017, The Haven, Wolverhampton, Neontribe and women who had been through domestic abuse co-designed a web app to encourage other women in
+            that situation to seek support and advice. We want to find out if the structure of that app could be used by different organisations. We believe
+            that reassuring people that their fears and challenges are shared could help them take early steps on a journey of change such as contacting a support service. """
+            ]
+      }
+    , { id = 2
       , name = "Bite size information"
-      , slug = "bitesize-information"
+      , slug = "bite-size-info"
       , icon = "information"
       , infoText =
-            [ """We include bite size information relating to a small number of fears and barriers that are likely to be stopping people from taking the
-            next step. Keeping the language simple we aim to be open and honest, with a reassuring tone. We resist the temptation to provide all the
-            information people might need. Our goal is to offer just enough to help people reach out to support services. Other websites have excellent
-            longer content."""
+            [ """We include bite size information relating to a small number of fears and barriers that are likely to be stopping people from taking the next step. Keeping the
+            language simple we aim to be open and honest, with a reassuring tone. We resist the temptation to provide all the information people might need. Our goal is to offer just enough to
+            help people reach out to support services. Other websites have excellent longer content. """
             ]
       }
-    , { id = 2
+    , { id = 3
       , name = "Illustrations"
-      , slug = "illustrations"
+      , slug = "Illustrations"
```

Comment:
keep the little i - urls are better all lowercase and this turns up in the url (I think)